lib/lita/version.rb

```diff
@@ -1,4 +1,4 @@
 module Lita
   # The current version of Lita.
-  VERSION = "4.6.0"
+  VERSION = "4.6.2"
```

Comment:
Don't change the version. The master branch is targeting Lita 5 now. This wouldn't be a patch-level version change anyway since it introduces new stuff.